Refactor of contribute pages (#1437)
elixir-europe/rdmkit: pages/contribute/how_to_contribute.md

---
title: How to contribute to RDMkit
toc: false
---

This project is only possible with the many [community contributors](https://rdmkit.elixir-europe.org/contributors). RDMkit is an open project where everyone can contribute to the site.

> If you wish to contribute to this community-led project, we ask you to remember the following for better consistency and efficiency: 

Reviewer: > If you wish to contribute to this community-led project, we ask you to remember the following for better consistency and efficiency:

If you wish to contribute, please:

Maybe we can keep this shorter, and not repeat that it is a community project.

> If you wish to contribute to this community-led project, we ask you to remember the following for better consistency and efficiency: 

Reviewer 2: ```suggestion
If you wish to contribute, please remember the following for better consistency and efficiency: 
```

* Follow our [style guide](https://rdmkit.elixir-europe.org/style_guide).
* Ensure your content respects copyright. Please follow our[ Copyright guidelines](https://rdmkit.elixir-europe.org/copyright).
* Provide acknowledgement for all your contributions.

## Ways of contributing

> * Google Doc. If you are unfamiliar with Git and prefer to use a classic document editor, follow [the Google Doc way](https://rdmkit.elixir-europe.org/google_doc_way)

Reviewer: > ...and prefer to use a classic document editor,...

...and prefer to use a Word-like editor,... ?

Wasn't sure what a "classical" document editor was. QuarkXPress is also a document editor with a long history, but nothing like Google Docs.

Author: I agree with docs and started changing changed it further at some places accordingly

> * Google Doc. If you are unfamiliar with Git and prefer to use a classic document editor, follow [the Google Doc way](https://rdmkit.elixir-europe.org/google_doc_way)

Reviewer 2: ```suggestion
* Google Docs. If you are unfamiliar with Git and prefer to use a classic document editor, follow [the Google Docs way](https://rdmkit.elixir-europe.org/google_doc_way)
```

* GitHub. If you are unfamiliar with Git but want to try contributing on GitHub with no technical knowledge required, follow [our step-by-step instructions](https://rdmkit.elixir-europe.org/github_way)
* Git. If you are familiar with Git, fork the repo and create a pull request following our [instructions](https://rdmkit.elixir-europe.org/working_with_git)

## Acknowledgement and ownership of content

> Contributors will be shown at the bottom of each page and on the main [contributors page](https://rdmkit.elixir-europe.org/contributors). if listed in the metadata of the markdown file. All contributors will also be displayed on the. If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).

Reviewer 2: > All contributors will also be displayed on the.

This sentence does not seem to be complete

Reviewer: I'm guessing that's a fragment that shouldn't be there. Maybe the passage should read:

> Contributors will be shown at the bottom of each page and on the main [contributors page](https://rdmkit.elixir-europe.org/contributors) if listed in the metadata of the markdown file. If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).

Reviewer 2: > I'm guessing that's a fragment that shouldn't be there.

I think you are right.

> If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).

I never realised adding your information to the CONTRIBUTORS yaml was not mandatory. I think we should require some information (at least the ORCID) since we are promoting best practices. Any thoughts @bedroesb?

Author: Can we assume everyone has a ORCID?

Reviewer 2: > Can we assume everyone has an ORCID?

It would be my assumption, but maybe I am living inside a bubble. ORCID is typically required by scientific journals to publish, so at least there is a strong push for scientists to create one anyway. (I created mine 7 years ago)

> Contributors will be shown at the bottom of each page and on the main [contributors page](https://rdmkit.elixir-europe.org/contributors). if listed in the metadata of the markdown file. All contributors will also be displayed on the. If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).

Reviewer 2: ```suggestion
Contributors will be shown at the bottom of each page and on the main [contributors page](https://rdmkit.elixir-europe.org/contributors) if listed in the metadata of the markdown file. If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).
```

title: How to contribute to RDMkit
---

This project is only possible with the many [community contributors](contributors). RDMkit is an open project where everyone can contribute to the site.

If you wish to contribute, please remember the following for better consistency and efficiency: 

* Follow our [style guide](style_guide).
* Ensure your content respects copyright. Please follow our[ Copyright guidelines](copyright).
* Provide acknowledgement for all your contributions.

## Ways of contributing

<div class="row row-cols-1 row-cols-md-2 row-cols-lg-3 g-4 ways-to-contribute text-center mt-5">
  <div class="col">
    <div class="card bg-light h-100">
      <img src="{{ '/assets/img/section-icons/github.svg' | relative_url }}" class="card-img-top h-icon-6 pt-3" alt="GitHub Icon">
      <div class="card-body">
        <a href="{{ 'github_way' | relative_url }}" class="stretched-link">
          <h3 class="card-title text-dark mt-0">GitHub</h3>
        </a>

>         <p class="card-text">If you are unfamiliar with Git but want to try contributing on GitHub with no technical knowledge required.</p>

Reviewer 2: ```suggestion
        <p class="card-text">If you are unfamiliar with Git but want to contribute on GitHub. No technical knowledge is required.</p>
```
